src/references/index.ts

```diff
@@ -161,6 +162,7 @@ export const NATIVE_ASSETS_PER_CHAIN: Record<ChainId, AddressOrEth> = {
   [ChainId.blastSepolia]: AddressZero,
   [ChainId.polygonAmoy]: AddressZero,
   [ChainId.degen]: AddressZero,
+  [ChainId.apechain]: AddressZero,
```

Comment:
i think NATIVE_ASSETS_PER_CHAIN can be removed now and use `chainsNativeAsset` 

import { chainsNativeAsset } from '@/chains';